app/views/common/_external_logout.html.erb

```diff
@@ -0,0 +1,2 @@
+<div id='external_logout'></div>
+<%= mount_react_component('ExternalLogout', '#external_logout', props.to_json, flatten_data: true)%>
```

Comment:
Does having this in a partial bring us anything when compared with putting it directly into `app/views/users/extlogout.html.erb`?

```erb
<div id='external_logout'></div>
<%= mount_react_component('ExternalLogout', '#external_logout', external_logout_props.to_json, flatten_data: true)%>
```

Reply:
I can put it inside the `external_logout_helper.rb` file, if that's what you meant ?
Currently this is the way we mounted several other react components, I know @ezr-ondrej was investigating another ways.

Reply:
I meant putting it directly into the view and omit this partial and `mount_external_logout` helper, but I have no strong opinion. But you are right we do it the same way in several other places.

Reply:
Because it's the only place that mounting  `ExternalLogout` i'm in favor to move `mount_react_component` into `extlogout.html.erb `